api/src/main/java/org/eclipse/microprofile/rest/client/package-info.java

```diff
@@ -0,0 +1,30 @@
+/*
+ *******************************************************************************
+ * Copyright (c) 2016-2017 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+
+/**
+ * <p>CDI Support for Microprofile Fault Tolerance
```

Comment:
another reference to Fault Tolerance.  :)  other than that, it looks good.  I'll approve to avoid delays in merging.

Reply:
I'd prefer to see this fixed, as there's a few other unneeded changes.